src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1454,6 +1455,10 @@ private function getErrorMessages(Exception $e)
                 'Only numbers and decimal points (".") are allowed in the amount fields, e.g. 10.50 or 1050.',
                 'Admin.Orderscustomers.Notification'
             ),
+            InvalidCartRuleValueException::class => $this->trans(
+                'It looks like the value is invalid: <ol><li>Percent or amount value must be greater than 0.</li><li>Percent value cannot exceed 100.</li><li>Amount value cannot exceed the total price of this order.</li></ol>',
```

Comment:
I validate the PR but I'm not a fan of embedding html inside translation chains 🤔

Reply:
I know the PR has already been validated, but wouldn't it be better to manage dedicated exception codes for each use case, this would allow to have a precise message for each context
And it removes the html from wording which is really not a good practice

Reply:
At least if we don't manage each use case with an exception we can break this wording into three to avoid having html in the wording:

```php
sprintf('%s<ol><li>%s</li><li>%s</li></ol>',
                $this->trans(
                    'It looks like the value is invalid:',
                    'Admin.Orderscustomers.Notification'
                ),
                $this->trans(
                    'Percent or amount value must be greater than 0.',
                    'Admin.Orderscustomers.Notification'
                ),
                $this->trans(
                    'Amount value cannot exceed the total price of this order.',
                    'Admin.Orderscustomers.Notification'
                )
            )
```

Reply:
@jolelievre done 🙂